pkg/reconciler/apis/permissionclaimlabel/permissionclaimlabel_reconcile.go

```diff
@@ -224,7 +240,7 @@ func (c *controller) reconcilePermissionClaims(ctx context.Context, apiBinding *
 				if err != nil {
 					return err
 				}
-				delete(newObjectMeta.GetLabels(), claim.key)
+				// Empty Patch, will relay on the admission to add the resources.
 				err = c.patchGenericObject(ctx, oldObjectMeta, newObjectMeta, gvr, lc)
```

Comment:
please simplify patchGenericObject to actually send just an empty patch. It looks like it still does heavy logic now.

Reply:
updated